app/scripts/controllers/serviceInstance.js

```diff
@@ -120,7 +120,14 @@ angular.module('openshiftConsole')
       var updateSchema = _.get($scope.plan, 'spec.instanceUpdateParameterSchema');
       var planUpdatable = (_.size(_.get(updateSchema, 'properties')) > 0) || (_.get($scope.serviceClass, 'spec.planUpdatable') && (_.size($scope.servicePlans) > 1));
 
-      $scope.editAvailable = planUpdatable && serviceInstanceReady($scope.serviceInstance) && !_.get($scope.serviceInstance, 'metadata.deletionTimestamp');
+      $scope.editAvailable =
+        (updateSchema || planUpdatable) &&
```

Comment:
@jeff-phillips-18 I think there was a bug here before. It should be editable sometimes even if the plan is not updatable, correct?